### ISSUE TITLE
Support for Global Hotkey inside VMs

### ISSUE THREAD

**Issue by notoriousturtle** [MacPass/MacPass#797]
**Is your feature request related to a problem? Please describe.**
The current global hotkey does not work inside VMs. I'd like it to do so.

**Describe the solution you'd like**
Make the global hotkey work inside of a VM running via VMware Fusion.

**Describe alternatives you've considered**
Copy and paste.

**Additional context**
No.

**Comment by mstarke** [comment]
Does Autotype work for your setup or can you only use copy and paste?

**Comment by notoriousturtle** [comment]
Yeah, Autotype is working for me, at least in Chrome. I assume it only supports browsers?

I'm running MacOS as my base OS, and commonly use a Windows, or another MacOS VM. It would be handy if Autotype worked in browsers inside those VMs, from the base OS if you know what I mean?

Its not a feature I crucially want, and am not sure how much I'd use it, but figured it would be handy, and maybe more useful for others.

**Comment by mstarke** [comment]
Please note the difference between Global Autotype and Autotype. Autotype requires you to select an entry an then trigger Autotype. MacPass then brings back the previous application and types the details. Global Autotype tries to remove the need to got to MacPass and select the entry by trying to find the correct entry using the window title of the current application.

Both systems aren't app specific. The idea is to use the window title so any application can benefit from the system. Mostly it's used inside browsers since most user accounts are manages inside a browser, however I for example use Autotype for a lot of other occasions e.g. the AppStore.

For Global Autotype to work inside a browser in a VM the VM Window needs to reflect the browser window title for MacPass to get a meaningful match. Autotype on the other hand should work all the time but you have to switch to MacPass, select the desired entry and then perform the Autotype.

**Comment by notoriousturtle** [comment]
Ah, I didnt realise there were two different Autotypes. 

Sounds like I just need to name my entries a bit cleaner to get Global Autotype to work. I quickly tested Autotype (not global) inside a VM running Windows, in Notepad. It did not print the password. It did print the username however. Have you tested this by any chance?
